### DISCUSSION TITLE
Call for Copilot CLI

### DISCUSSION THREAD

**Discussion by tszdanger** [community/community#19329] [Copilot Conversations] [answered]
Hi,

I wonder whether we can call Copilot in CLI? I could not find enough material/sources for this. It seems that we can only use it within an IDE, is that true?

**Comment by davecheney** [comment, marked as answer]
> It seems that we can only use it within an IDE, is that true?

That is correct.

**Reply by tszdanger** [reply]
Thanks for your reply. May I ask if you have any plans about providing such a CLI?
